menu/menu_setting.c

```diff
@@ -3327,6 +3327,34 @@ static bool setting_append_list(
                   general_write_handler,
                   general_read_handler,
                   SD_FLAG_NONE);
+				  
+				  
+			CONFIG_BOOL( //ben
```

Comment:
please follow the indentation style from the rest of the code.
We use 3 spaces for tabs.